ui-v2/app/components/tabular-collection.js

```diff
@@ -85,9 +79,10 @@ const change = function(e) {
       // 'actions_close' would mean that all menus have been closed
       // therefore we don't need to calculate
       if (e.currentTarget.getAttribute('id') !== 'actions_close') {
-        const $tr = closest('tr', e.currentTarget);
-        const $group = sibling(e.currentTarget, 'ul');
-        const $footer = [...$$('footer[role="contentinfo"]')][0];
+        const dom = get(this, 'dom');
+        const $tr = dom.closest('tr', e.currentTarget);
+        const $group = dom.sibling(e.currentTarget, 'ul');
```

Comment:
Yuk, how did I ever think this was a good idea! 😄 

Actually I need to have a better look and refresh my memory on these utils, maybe there was a reason, but I doubt it!